src/kOS/Suffixed/VectorRenderer.cs

```diff
@@ -444,14 +444,16 @@ public void RenderPointCoords()
                 useWidth = (float)(Width * Scale * mapWidthMult);
 
                 // Position the arrow line:
-                line.SetVertexCount(2);
-                line.SetWidth(useWidth, useWidth);
+                line.positionCount = 2;
+                line.startWidth = useWidth;
+                line.endWidth = useWidth;
                 line.SetPosition(0, point1);
                 line.SetPosition(1, point2);
 
                 // Position the arrow hat:
-                hat.SetVertexCount(2);
-                hat.SetWidth(useWidth * 3.5f, 0.0F);
+                hat.positionCount = 2;
+                hat.startWidth = useWidth * 3.5f;
+                hat.endWidth = 0.0f;
```

Comment:
Fixes Warnings: obsoleted methods - SetVertex and SetWidth.

Unity changed the API a bit here.